src/request.js

```diff
@@ -217,7 +217,8 @@ class Request {
                     ? errorOrMessage.message
                     // .stack includes the message
                     : errorOrMessage.stack;
-            } else if (errorOrMessage.message) {
+            } else if (errorOrMessage.hasOwnProperty('message')) { // eslint-disable-line no-prototype-builtins
```

Comment:
Here I believe the ESLint warning is a valid one. We're transforming various errors into a standard Error, but those errors could have been created without the `Object` prototype and this would fail. Better use the more verbose, but safer:

```javascript
Object.prototype.hasOwnProperty.call(errorOrMessage, 'message')
```

Reply:
It was overkill anyway. It was a local setup issue, I persuaded my IDE not to be confused by accessing unresolved variables. Should be tried in VSCode first.

Reply:
This change has been reverted now.